hist/hist/src/TH1.cxx

```diff
@@ -566,6 +567,9 @@ TH1::TH1(): TNamed(), TAttLine(), TAttFill(), TAttMarker()
    fXaxis.SetParent(this);
    fYaxis.SetParent(this);
    fZaxis.SetParent(this);
+   ResetBit(TH1::kAutoBinPTwo);
```

Comment:
What is the purpose of resetting them here?

Related question.  Are those bits intended to be persistent or transient?